.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: Format Files
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  format:
```

Comment:
Without the `permissions` field I'm not sure that this is going to be able to do anything